lib/configs/api-gateway/schema.ts

```diff
@@ -479,7 +479,7 @@ export const articleListSchema: JsonSchema = {
                     items: {
                         type: JsonSchemaType.OBJECT,
                         properties: {
-                            "category": {
```

Comment:
The json schema will be invalid if you delete this line since its a property name. Check https://json-schema.org/specification.html

Reply:
I have set it to "type", now I changed back.